Calibration/HcalCalibAlgos/test/HcalCorrPFCalculation.cc

```diff
@@ -187,24 +185,19 @@ void HcalCorrPFCalculation::analyze(edm::Event const& ev, edm::EventSetup const&
     edm::LogWarning("CalibConstants") << "   Not Found!! ";
   }
 
-  edm::Handle<HBHERecHitCollection> hbhe;
-  ev.getByToken(tok_hbhe_, hbhe);
+  const edm::Handle<HBHERecHitCollection>& hbhe = ev.getHandle(tok_hbhe_);
   const HBHERecHitCollection Hithbhe = *(hbhe.product());
 
-  edm::Handle<HFRecHitCollection> hfcoll;
-  ev.getByToken(tok_hf_, hfcoll);
+  const edm::Handle<HFRecHitCollection>& hfcoll = ev.getHandle(tok_hf_);
   const HFRecHitCollection Hithf = *(hfcoll.product());
 
-  edm::Handle<HORecHitCollection> hocoll;
-  ev.getByToken(tok_ho_, hocoll);
+  const edm::Handle<HORecHitCollection>& hocoll = ev.getHandle(tok_ho_);
   const HORecHitCollection Hitho = *(hocoll.product());
```

Comment:
Isnt there a getData method for these? is case the handle is not used for the validity check, and the getData exists, this could be further simplified